app/assets/javascripts/to_do_items.coffee

```diff
@@ -40,14 +41,56 @@ class @TodoItem
   detectCancelFormButton = (id) ->
     $("#cancel-item-button-#{id}")
 
+  extractId = (str) ->
+    str.split('-').pop()
+
   removeCallout = (listId) ->
     detectList(listId).find('div.bs-callout').remove()
 
+  saveOnClick = (item) ->
+    $(item).click () ->
+      updateItem extractId(item['id']), item['value']
+
+  markAsDone = (id, listId) ->
+    completeList = detectList(listId).find('ul.complete')
+    detectItem(id).appendTo completeList
+
+  unmark = (id, listId) ->
+    incompleteList = detectList(listId).find('ul.incomplete')
+    detectItem(id).appendTo incompleteList
```

Comment:
@vlasiak, I think both `markAsDone` and `unmark` methods can be merged into the one as they have the same logic. I would name it `toggle` and based on item `done` attributed used proper css style. Also, that will simplify the body of the callback where given methods are being used.  Does it make a sense?